.kiro/specs/2. offline-first-sync/tasks.md

```
@@ -0,0 +1,661 @@
+      -- only visible / mutable by the owning user or by a trusted DB role.
+      -- Replace '<app_db_role>' with the name of your application DB role (for
+      -- example a role that your Edge Functions or trusted services switch to).
+      -- The policy below allows actions when the authenticated jwt user matches
+      -- the row's user_id OR when the request context indicates a trusted role.
+
+      -- Enable RLS on the table
+      ALTER TABLE public.sync_idempotency ENABLE ROW LEVEL SECURITY;
+      ```
+
+    -- Allow SELECT only for the owning user or trusted roles
+    /_ NOTE: PostgreSQL does NOT support `CREATE POLICY IF NOT EXISTS`.
```

markdownlint-cli2 (0.17.2)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)